quickstart/docker-compose.yaml

```diff
@@ -21,22 +22,26 @@ services:
       KAFKA_CFG_AUTO_CREATE_TOPICS_ENABLE: "true"
 
   kafka-init:
-    image: docker.io/bitnami/kafka:latest
+    image: docker.io/bitnami/kafka:3.2
+    container_name: temp-kafka-init-zq
     command: 
       - "/bin/bash"
       - "-c"
       -  | 
         /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic items-crud --config cleanup.policy=compact
         /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic events-sse
```

Comment:
We often use compact for sse so that replay gets the latest of each key on reconnect, but that depends on the message keys being unique per "object" we want to observe. Consider using compact if it makes sense for the usecase.